### ISSUE TITLE
MathJax 4.0 possible issue

### ISSUE THREAD

**Issue by glarose** [openwebwork/webwork2#2828]
I have been working with the staff in my accessibility office, and they had the following observation about changes made in MathJax v4.  They note that it has many improvements and useful accessibility features, but expressed some concern about its default configuration.

The underlying change that concerned them is that by default it does not work with MathCAT (in NVDA or JAWS) or other assistive technologies. In particular, the default configurations of MathJax v4 no longer include the underlying hidden MathML for the math expressions it renders.  The change seems to be that in MathJax v2 and v3, the assistive-mml extension was loaded and enabled by default, but in v4, the explorer component is the default assistive tool.  The MathJax documentation says that the explorer can be disabled and the assistive MathML re-enabled using the MathJax contextual menu on any typeset expression, and page
author’s can override the defaults in their MathJax configuration objects, if they so desire.  (As an aside, the staff at my end say that actually one has to turn off the MathJax semantic enrichment option, not the the explorer on its own; turning off semantic enrichment also disables the MathJax explorer.)

I don't think this changes WeBWorK's accessibility, but may change how we want to present it to users.  The people I'm working with think that the default really should be to have MathJax set to allow the user's screen reader to handle accessibility, and to allow them to go over to MathJax instead if desired.  If we think that's the right strategy, we might just change the default MathJax configuration in WeBWorK.  If we go with the default in MathJax v4, we may want to update the orientation set.

**Comment by somiaj** [comment]
Is this issue worth keeping open? We now are on mathjax 4.1, and it sounds like screen readers should correctly deal with its defaults.

**Comment by somiaj** [comment]
Closing for now, if there is a specific issue with mathjax accessibility it should be reported to mathjax. There is an issue to update student orientation, maybe this can address any potential issues.
